### ISSUE TITLE
Talkback screen reader not reading Capacitive buttons on Samsung Captivate Glide SGH-I927

### ISSUE THREAD

**Issue by GoogleCodeExporter** [clc/eyes-free#269]
```
What steps will reproduce the problem?
1. Run finger along bottom of touchscreen over capacitive buttons on Samsung 
Captivate Glide SGH-I927.
2. Talkback does not announce any buttons such as back, home, etc.
3 This has occurred in all Android versions from 4.04 to now 4.2.2, Talkback 
versions 3.1 now to 3.3.

What is the expected output? What do you see instead?
Expected output, Talkback should announce what each capacitive button does. 
Talkback instead only gives vibration feedback no voice feedback as to which 
button is being accessed.

What version of the product are you using? On what operating system?
Currently am using Talkback 3.3 on Jellybean running Cyanogenmod 10.1. I have 
also seen this behavior on AT&T stock ICS firmware running Talkback 3.1. Note, 
on a Galaxy S3 running stock Jellybean firmware this issue did not occur, 
talkback 3.3 updated from playstore read out which capacitive button was being 
accessed.

Please provide any additional information below.


```

Original issue reported on code.google.com by `davespho...@gmail.com` on 30 May 2013 at 6:27

**Comment by GoogleCodeExporter** [comment]
```
TalkBack is not able to provide feedback for hardware controls
```

Original comment by `caseybur...@google.com` on 5 Dec 2013 at 10:00
- Changed state: **Invalid**
